CMakeLists.txt

```diff
@@ -396,11 +396,11 @@ endif ()
 # ----------------------------------------------------------------------------
 # installation directories
 if (OS_WINDOWS)
-  set (RUNTIME_INSTALL_DIR Bin)
-  set (LIBRARY_INSTALL_DIR Lib)
-  set (INCLUDE_INSTALL_DIR Include)
-  set (CONFIG_INSTALL_DIR  CMake)
-  set (PKGCONFIG_INSTALL_DIR)
+  set (RUNTIME_INSTALL_DIR "bin")
+  set (LIBRARY_INSTALL_DIR "lib")
+  set (INCLUDE_INSTALL_DIR "include")
+  set (CONFIG_INSTALL_DIR  "share/${PACKAGE_NAME}")
```

Comment:
Well, what you've cited above actually only applies to macOS and Linux (the U stands for Unix...). CMake's `find_package` uses different paths for Windows:

```
<prefix>/                                                       (W)
<prefix>/(cmake|CMake)/                                         (W)
<prefix>/<name>*/                                               (W)
<prefix>/<name>*/(cmake|CMake)/                                 (W)
<prefix>/<name>*/(lib/<arch>|lib|share)/cmake/<name>*/          (W/U)
<prefix>/<name>*/(lib/<arch>|lib|share)/<name>*/                (W/U)
<prefix>/<name>*/(lib/<arch>|lib|share)/<name>*/(cmake|CMake)/  (W/U)
```
but that includes `<prefix>/lib/cmake/<name>`.

It's just that when we already change the names to lowercase and include the package name in the config path (although on Windows I assume it is already part of the `<prefix>`, e.g., `C:\Program Files\gflags`) and thereby match the installation hierarchy on Unix, I would probably prefer to then have the same paths also for the CMake configuration files (besides `<prefix>` obviously).